### DISCUSSION TITLE
part-ii/2020-4/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#126] [Question comments]
Part II 2020 Algebraic Topology: Paper 1, Section II, $21 \mathbf{F}$ 

https://questions.tripos.org/part-ii/2020-4/

**Comment by eha1234** [comment]
For part (c), it is helpful to consider subgroups of index 2 as kernels of group homomorphisms to Z/2. Then classify the possible covering spaces.
